wandb/old/retry.py

```diff
@@ -109,9 +112,13 @@ def __call__(self, *args, **kwargs):
                     raise
                 if self._num_iter == 2:
                     logger.exception('Retry attempt failed:')
-                    wandb.termlog(
-                        '{} ({}), entering retry loop. See {} for full traceback.'.format(
-                            self._error_prefix, e.__class__.__name__, util.get_log_file_path()))
+                    if isinstance(e, HTTPError):
+                        self.retry_callback(e.response.status_code, e.response.text)
+                    else:
+                        # todo: would like to catch other errors, eg wandb.errors.Error, ConnectionError etc
+                        # but some of these can be raised before the retry handler thread (RunStatusChecker) is
+                        # spawned in wandb_init
+                        wandb.termlog('{} ({}), entering retry loop.'.format(self._error_prefix, e.__class__.__name__))
```

Comment:
eg when wandb can't connect to the server on startup we should print ConnectionErrors to terminal, but since RunStatusChecker only starts on run start, processing via retry_callback would result in hanging w/o any message

Reply:
yep, We will fix this in another way, this is part of the message polling changes.